plugin/pkg/scheduler/api/v1/types.go

```diff
@@ -100,3 +108,54 @@ type LabelPreference struct {
 	// If false, higher priority is given to nodes that do not have the label
 	Presence bool `json:"presence"`
 }
+
+// Holds the parameters used to communicate with the extender. If a verb is unspecified/empty,
+// it is assumed that the extender chose not to provide that extension.
+type ExtenderConfig struct {
+	// URLPrefix at which the extender is available
+	URLPrefix string `json:"urlPrefix"`
+	// Verb for the filter call, empty if not supported. This verb is appended to the URLPrefix when issuing the filter call to extender.
+	FilterVerb string `json:"filterVerb,omitempty"`
+	// Verb for the prioritize call, empty if not supported. This verb is appended to the URLPrefix when issuing the prioritize call to extender.
+	PrioritizeVerb string `json:"prioritizeVerb,omitempty"`
```

Comment:
It would be helpful to know the precedence of `FilterVerb` and `PrioritizeVerb` relative to one another